src/Common/FitData.m

```diff
@@ -94,17 +94,17 @@
         setappdata(h,'canceling',0)
     end
 
-    if (isempty(h)), j_progress('Fitting voxel ',l); end
-    for ii = 1:l
+    if (isempty(h)), fprintf('Fitting %d voxels.\n',numVox); end
```

Comment:
@notZaki Is there any reason in particular as to why you wanted us to stop using the custom function [`j_progress()`](https://github.com/qMRLab/qMRLab/blob/07cd6ff2b5faedc5c1bc0bf74024dad40603e432/src/Common/tools/j_progress.m) and instead opt for `fprintf()` at this particular line? It is an old function, I was just wondering if you encountered any particular issue that prompted this change.

Reply:
It's because `j_progress()` will print out:
```
Fitting voxel        0/<NumberOfVoxels>
```
This counter is meant to be updated in the loop, but the `j_progress()` in the loop is disabled.
So the user only sees the above message once and then nothing else.

It would've been better if I had just commented out the line instead of removing it---this will be fixed in next batch of commits.

Reply:
Reverted in 787a01abb0c9da4

I think either one works, but since this is unrelated to the core of this PR, let's just keep it the same for now. If you had any specific concerns with `j_progress`, or simply want to propose all `j_progress` calls to `fprintf`, please open an issue and then we can reach out to the dev who initially chose this function over `fprintf` to see what the benefit of `j_progress` was.

Reply:
Ahh, just saw your comment @notZaki 

Yeah, we should actually just remove the counter altogether. This only shows up for command line calls, is very slow, and in Jupyter Notebooks creates very (very) long outputs.

Reply:
Switched to fprintf and changed message (no voxel #) in bd69e1.